tests/browser/test/test_setup.js

```diff
@@ -200,6 +200,22 @@ async function connect(
   await draggedBlock.dragAndDrop(delta);
 }
 
+async function dragNthBlockFromFlyout(browser, categoryName, n, x, y) {
+  const flyoutBlock = await getNthBlockOfCategory(browser, categoryName, n);
+  await new Promise((resolve) => setTimeout(resolve, 2000)); // 2 sec
```

Comment:
We can actually use `await browser.pause(ms)` instead! I'm investigating the `element.waitUntil` methods as well.

Reply:
Switched to `browser.pause`.